coordinator/src/routes/tracker/task.ts

```diff
@@ -8,34 +8,66 @@ import {getScope, RequestHeaders} from "../../utils/headers"
 import * as LosslessJson from "lossless-json"
 import {isBundle, isTask} from "../../utils/type-guards"
 import {validateQueryParameters} from "../../services/validation/query-validator"
+import {getCodeableConceptCodingForSystem, toArray} from "../../services/translation/common"
+import moment from "moment"
 
-export enum QueryParam {
-  IDENTIFIER = "identifier",
-  FOCUS_IDENTIFIER = "focus:identifier",
-  PATIENT_IDENTIFIER = "patient:identifier"
+export interface ValidQuery {
+  "identifier"?: string
+  "focus:identifier"?: string
+  "patient:identifier"?: string
+  "business-status"?: string
+  "authored-on"?: string | Array<string>
 }
 
-export type ValidQuery = Partial<Record<QueryParam, string>>
-
-interface QueryParamProperties {
+export interface QueryParamDefinition {
+  name: keyof ValidQuery,
+  supportedBySpine: boolean
```

Comment:
Needs a better name. I want something which means "Spine allows queries using this as the only parameter", but the current name suggests something more like "Spine understands this parameter".

Reply:
`validInIsolation`?